people/francesco-perera.md

```diff
@@ -0,0 +1,7 @@
+Francesco Perera
+------------------
+
+![](photos/francescoperera.jpg)
```

Comment:
Please rename this file to francsco-perera.jpg to be consistent with the rest of the file names.